### ISSUE TITLE
Update typings after version 1.18.0

### ISSUE THREAD

**Issue by Kenny407** [ConvertAPI/convertapi-node#31]
Hello, 

Thanks for the recent update of the package. 

I have noticed that the `convert` returns a `Promise<Result>` but `Result` does not contain `JobId` which can be returned when calling `convert` with the `Async` flag.

Here's an example:

```ts
const response = await client.convert('pdf',
      {
        File: url,
        WebHook: 'my webhook url',
        Async: true,
      }
    )
```
The output of this response contains a lot of parameters that are not included in `Result` 

here's the log of the previous statement.
```ts
 {
  api: { 
    secret: '*******',
    baseUri: 'https://v2.convertapi.com',
    conversionTimeout: undefined,
    conversionTimeoutDelta: 10,
    uploadTimeout: 1800,
    downloadTimeout: 1800,
    userAgent: 'ConvertAPI-Node/1.8.0',
    proxy: undefined,
  },
  response: { JobId: 'SOME-JOB-ID-GOES-HERE' }
}
```

Could you please support this? Also the location of the `index.d.ts` is not the correct one, you could put it in the root of the project or in a `types` folder.

Thanks in advance.

**Comment by tomasr78** [comment]
@Kenny407  The ConvertAPI Async is not supported in NodeJS because the programming language is asynchronous by default and it makes no sense to use rest api async methods.

**Comment by tomasr78** [comment]
@laurynas-baltsoft Could you please fix TS location, please?

**Comment by Kenny407** [comment]
For my use case it does make sense to use the async API methods even with NodeJS. Because I do not want to wait for the conversion to be done sequentially.

Quick example:
1. I need to create a batch of 10 conversions (without having to wait one by one to be finished).
2. I check (use the polling part of the API) when the jobs are done. 

Given what I said, is it possible to include it?

It could be an optional property of the `Result` interface.

**Comment by tomasr78** [comment]
@laurynas-baltsoft Could you please create a demo with async batch conversion to show @Kenny407 how parallel conversions can be executed without blocking each other. Also would be cool if you could introduce `MaxDegreeOfParallelism` property in that demo to let set the maximum number of concurrent tasks.

**Comment by laurynas-baltsoft** [comment]
@Kenny407 I have added raw `response` and `jobId` accessors to the response object:
https://github.com/ConvertAPI/convertapi-node/pull/32/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8L17

Also adjusted `index.d.ts` location. Thanks for the tip.

Released `1.9.0` version.
